src/test/openads/domain/service/LoggerInitializerTest.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
This line should not be necessary anymore... ^_^
![](https://media.giphy.com/media/DfbpTbQ9TvSX6/giphy.gif)